src/NLog/Targets/WebServiceTarget.cs

```diff
@@ -250,6 +250,45 @@ protected override void DoInvoke(object[] parameters, AsyncContinuation continua
             }
         }
 
+        /// <summary>
+        /// Builds the URL to use when calling the web service for a message, depending on the WebServiceProtocol.
+        /// </summary>
+        /// <param name="parameterValues"></param>
+        /// <returns></returns>
+        private Uri BuildWebServiceUrl(object[] parameterValues)
+        {
+            if (this.Protocol != WebServiceProtocol.HttpGet)
+            {
+                return this.Url;
+            }
+            
+            //if the protocol is HttpGet, we need to add the parameters to the query string of the url
+            var queryParameters = new StringBuilder();
+            string separator = string.Empty;
+            int i = 0;
+            foreach (MethodCallParameter parameter in this.Parameters)
```

Comment:
Looks good, but I think `for` is better if we already need an `i`. Do you agree?